package ai.knowly.langtorch.store.vectordb.integration.pgvector.exception;

public class PGVectorServiceInstantiationException  extends  RuntimeException {

    public PGVectorServiceInstantiationException(Throwable cause) {
        super(cause);
    }
}
